git-rebase-via-merge.sh

```diff
@@ -6,7 +6,7 @@
 # Copyright (c) 2022 Baur Atanov
 #
 
-default_base_branch="origin/main"
+default_base_branch="origin/develop"
```

Comment:
Most developers need to rebase onto `develop` as I believe.

Reply:
Fair! (My workplace doesn't use git-flow, so I often forget about that)

Will fix!